src/webcomponents/disease-panel/disease-panel-grid.js

```diff
@@ -460,6 +463,9 @@ export default class DiseasePanelGrid extends LitElement {
                                     <a data-action="edit" class="btn force-text-left ${isAdmin ? "" : "disabled"}">
                                         <i class="fas fa-edit icon-padding" aria-hidden="true"></i> Edit ...
                                     </a>
+                                    <a data-action="edit" class="btn force-text-left ${OpencgaCatalogUtils.checkPermissions(this.opencgaSession.study, this.opencgaSession.user.id, this.permissionID) || "disabled" }">
+                                        <i class="fas fa-edit icon-padding" aria-hidden="true"></i> Edit ...
+                                    </a>
```

Comment:
It seems that the `Edit` action will be displayed twice in the actions dropdown of the Disease Panel Grid. Please, could you remove the old `Edit` action?